programs/building/README.md

```diff
@@ -4,22 +4,22 @@ An assortment of scarpet apps to help you with your creative builds. Some have g
 
 ## Creative building tools scarpet apps in alphabetical order with creator:
 
-### cover.sc
+### [cover.sc](https://github.com/gnembon/scarpet/blob/master/programs/building/cover.sc):
 #### By Firigion
 
 Lets you cover a set of blocks with a set of other blocks for ease of spawnproofing a decorative build. To use it, put the block to cover in your offhand and the block to cover it with in your main hand. To do a bunch of different blocks at a time, create pairs by placing a shulker box in the main hand and another one in the off hand. Pairs will be created from matching slots in the boxes. For more info, see [this](https://github.com/Firigion/scarpets#cover).
 
-### curves.sc
+### [curves.sc](https://github.com/gnembon/scarpet/blob/master/programs/building/curves.sc):
 #### By Firigion
 
 A tool to help you create helixes, spirals, waves, curvy circles stars _and much more_ (Idk what else, I'm probably forgetting something) parametrically. It comes with built in help command and interactive settings. If you need help or details on how stuff works, see [here](https://github.com/Firigion/scarpets#curves).
 
-### soft_replace.sc
+### [soft_replace.sc](https://github.com/gnembon/scarpet/blob/master/programs/building/soft_replace.sc):
 #### By Firigion
 
 Lets you replace blocks in an area with other blocks, preserving their block properties. This way, you can create a beautiful build out of slabs, stairs and log blocks and midway through the design process, change your pallette. See [here](https://github.com/Firigion/scarpets#soft-replace) for more info on how to use it.
 
-### shapes.sc
+### [shapes.sc](https://github.com/gnembon/scarpet/blob/master/programs/building/shapes.sc):
```

Comment:
The file name on github is "sshapes.sc", so this link does not work.

Reply:
```suggestion
### [sshapes.sc](https://github.com/gnembon/scarpet/blob/master/programs/building/sshapes.sc):
```